src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -886,7 +886,7 @@ private bool EmitFieldLoadReceiverAddress(BoundExpression receiver)
 
                 if (!field.IsStatic && EmitFieldLoadReceiverAddress(fieldAccess.ReceiverOpt))
                 {
-                    Debug.Assert(!field.IsVolatile, "volatile valuetype fields are unexpected");
+                    Debug.Assert(!field.IsVolatile, "volatile value type fields are unexpected");
```

Comment:
I'll change that. Which do you prefer?
Op 14 jul. 2015 19:52 schreef "Charles Stoner" <notifications@github.com>:

> In src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs
> <https://github.com/dotnet/roslyn/pull/3945#discussion_r34597659>:
>
> > @@ -886,7 +886,7 @@ private bool EmitFieldLoadReceiverAddress(BoundExpression receiver)
> >
> >                  if (!field.IsStatic && EmitFieldLoadReceiverAddress(fieldAccess.ReceiverOpt))
> >                  {
> > -                    Debug.Assert(!field.IsVolatile, "volatile valuetype fields are unexpected");
> > +                    Debug.Assert(!field.IsVolatile, "volatile value type fields are unexpected");
>
> Should be 'valuetype' or 'ValueType'.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotnet/roslyn/pull/3945/files#r34597659>.
>